charger/evsewifi.go

```diff
@@ -266,10 +267,19 @@ func (evse *EVSEWifi) currents() (float64, float64, float64, error) {
 	return float64(params.CurrentP1), float64(params.CurrentP2), float64(params.CurrentP3), err
 }
 
+// clear evse RFID
+func (evse *EVSEWifi) clearRfid() error {
+	url := evse.apiURL(evseClearRFID)
+	return evse.get(url)
+}
+
 // Identify implements the api.Identifier interface
 func (evse *EVSEWifi) identify() (string, error) {
 	params, err := evse.getParameters()
-	return *params.RFIDUID, err
+	if err != nil || *params.RFIDUID == "" {
+		return *params.RFIDUID, err
```

Comment:
Der Code war vorher schon falsch. Wenn err!=nil und params==nil dann crasht es :O

Reply:
Urspungsproblem in https://github.com/evcc-io/evcc/commit/a8137ec190b4b884e985383545699a5e039c4352 behoben